docs/adrs/0000-v2-setup-java.md

```diff
@@ -70,5 +72,5 @@ The `setup-java` action has some logic that creates a `settings.xml` file so tha
 
 - Users will have more flexibility and the freedom to choose a specific distribution that they would like (AdoptOpenJDK builds of OpenJDK in addition or Zulu builds of OpenJDK)
 - `setup-java` will be structured in such a way that will allow for more distributions to be easily added in the future
-- A large subset of users pin to `@main` or `@master` instead of to a specific version (`v1`). By introducing a breaking change by switching from `Zulu` to `AdoptOpenJDK` as the default, certain existing workflows for users might break
+- A large subset of users pin to `@main` or `@master` instead of to a specific version (`v1`). By introducing a breaking change that now requires a distribution to be specified, many users will have their workflows fail until they go and update their yaml
```

Comment:
We should update that only a small % of users pin to main and that we don't recommend it.